hs/src/Reach/Eval/Core.hs

```diff
@@ -3463,6 +3470,17 @@ evalPrim p sargs =
         (l, r) -> expect_ $ Err_xor_Types l r
       where
         prim = flip evalPrimOp sargs
+    SLPrim_mod -> do
+      (x, y) <- two_args
+      (x_ty, _) <- compileTypeOf x
+      (y_ty, _) <- compileTypeOf y
+      case (x_ty, y_ty) of
+        (T_UInt, T_UInt) -> evalPrimOp MOD sargs
+        (T_Bytes sz, T_UInt)
```

Comment:
You say in the docs it works on Digests, but there's no test for that and you don't allow it here

Reply:
See my initial PR comment 😉

Reply:
Sorry. I don't read :)